src/test/java/bio/terra/service/dataset/flight/inheritsteward/GetSnapshotIdsStepTest.java

```diff
@@ -41,13 +41,14 @@ class GetSnapshotIdsStepTest {
   @ValueSource(booleans = {true, false})
   void doStep(boolean inheritSteward) throws Exception {
     GetSnapshotIdsStep step =
-        new GetSnapshotIdsStep(snapshotDao, iamService, TEST_USER, DATASET_ID, inheritSteward);
+        new GetSnapshotIdsStep(snapshotService, iamService, TEST_USER, DATASET_ID, inheritSteward);
     FlightMap workingMap = new FlightMap();
     List<UUID> snapshotIds = Arrays.asList(SNAPSHOT_1, SNAPSHOT_2);
     when(flightContext.getWorkingMap()).thenReturn(workingMap);
 
     if (inheritSteward) {
-      when(snapshotDao.getSnapshotIds(DATASET_ID)).thenReturn(snapshotIds);
```

Comment:
I think this is the only non-test usage of this API, if so, the DAO method can be deleted. It was definitely added as part of this work. https://github.com/DataBiosphere/jade-data-repo/pull/1924

Also, the original PR was using `snapshotService.enumerateSnapshotIdsForDataset()`, and I made the suggestion to not use it.. 🤦